app/furniture/section_navigation/section_navigations/_section_navigation.html.erb

```diff
@@ -1,7 +1,17 @@
 <div class="grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4">
+  <% has_section_images = section_navigation.rooms.any? { |r| r.hero_image.present? } %>
```

Comment:
I don't love seeing this type of logic in templates because I like to swing toward logic-less templates generally, but...I also wouldn't elevate it to the model since it's quite view specific. Of course a ViewComponent class would be a nice place for this. (Just thinking out loud.)

Reply:
The other "Rails-y" place for this is a method in a helper -- we could start a `SectionNavigationHelper` and add this as a predicate there.

My feeling is that this probably wants to become a `ViewComponent` sooner rather than later, and in that case this will make sense as a method in there.

I generally avoid having complex logic in a view, but I also think that most views in the real world will have some kind of logic in them, so I'm not religious about it. In this case I feel like this is simple enough and enough of an area that we are likely still iterate on that this is ok for now.